proto_v2/epoch_v2.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+package fd.v2;
+
+import "slot_v2.proto";
+
+message SeedAddress {
+    /* The seed address base. (32 bytes) */
+    bytes base  = 1;
+
+    /* The seed path. (<= 32 bytes) */
+    bytes seed  = 2;
+
+    /* The seed address owner. (32 bytes) */
+    bytes owner = 3;
+}
+
+message AcctState {
```

Comment:
On a side-note, if we plan to add signature verification, we should probably include an optional signature in the account state as well